parmed/amber/netcdffiles.py

```diff
@@ -200,6 +200,7 @@ def id_format(filename):
         Remote NetCDF files cannot be loaded
         """
         if filename.startswith('http://') or filename.startswith('https://'):
+            or filename.startswith('ftp://') or filename.startswith('ftplib'):
```

Comment:
`ftplib` is a Python module, not a filename prefix.  Also, this is a `SyntaxError`.